Metallicity_Stack_Commons/valid_table.py

```diff
@@ -3,10 +3,12 @@
 from astropy.io import ascii as asc
 from astropy.table import Table, Column
 
-from .column_names import filename_dict, valid_table_names0  # , bin_names0, remove_from_list
+from .column_names import filename_dict, valid_table_names0
 
 
-def make_validation_table(fitspath: str):
+def make_validation_table(fitspath: str, vmin_4363SN, vmin_5007SN,
+                          vmax_4363sig, rlmin_4363SN,
+                          rlmax_4363sig, rlmin_5007SN):
```

Comment:
These inputs should be provided as optional keywords with default values (old values) set.

Reply:
I believe I fixed this